pkg/config/config_template.yaml

```diff
@@ -45,6 +45,22 @@ api_key:
 #   - env:prod
 #   - role:database
 
+# Split tags according to given string.
+# Supports host tags, container tags, excludes tags collected by the integration
```

Comment:
a few questions/nits on the doc wording:
1. customers should know what `host tags` are (we refer to them throughout the docs, and in-app there's a clear `host tags` section on every host of the infra list), but do we expect customers to know what `container tags` are exactly?
2. Instead of `Supports`/`excludes` I'd write `Only applies to`/`Does not apply to`, sounds more explicit to me :)
3. let's add `tags of dogstatsd metrics` to the "excluded" tags
4. grammar: plural: `integrations`